apps/fabric-website/package.json

```diff
@@ -37,7 +37,7 @@
     "es6-promise": "^4.1.0",
     "es6-weak-map": "^2.0.2",
     "highlight.js": "^9.12.0",
-    "office-ui-fabric-core": "^9.0.0",
+    "office-ui-fabric-core": "9.6.1-fluent2",
```

Comment:
This can update to `10.0.0`. If not this PR, it can be in a subsequent one. @jordandrako for FYI